### PR TITLE
Allow any string as Size parameter going forward. Add P5 in PSArgumen…

### PR DESCRIPTION
**Pull request by SiddharthChatrolaMs**
…tCompleter popup

## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [x] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

File: src/ResourceManager/RedisCache/ChangeLog.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+﻿<!--
```
Comment (cormacpayne) [inline src/ResourceManager/RedisCache/ChangeLog.md]: @SiddharthChatrolaMs this file isn't necessary -- the existing change log for RedisCache can be found [here](https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/RedisCache/Commands.RedisCache/ChangeLog.md).

File: src/ResourceManager/RedisCache/Commands.RedisCache/Commands/NewAzureRedisCache.cs [hunk 2]
```diff
@@ -41,10 +41,10 @@ public class NewAzureRedisCache : RedisCacheCmdletBase
         [ValidateNotNullOrEmpty]
         public string Location { get; set; }
 
-        [Parameter(ValueFromPipelineByPropertyName = true, Mandatory = false, HelpMessage = "Size of redis cache. Valid values: P1,P2, P3, P4, C0, C1, C2, C3, C4, C5, C6, 250MB, 1GB, 2.5GB, 6GB, 13GB, 26GB, 53GB")]
-        [ValidateSet(SizeConverter.P1String, SizeConverter.P2String, SizeConverter.P3String, SizeConverter.P4String,
+        [Parameter(ValueFromPipelineByPropertyName = true, Mandatory = false, HelpMessage = "Size of redis cache. Valid values: P1,P2, P3, P4, P5, C0, C1, C2, C3, C4, C5, C6, 250MB, 1GB, 2.5GB, 6GB, 13GB, 26GB, 53GB, 120GB")]
+        [PSArgumentCompleter(SizeConverter.P1String, SizeConverter.P2String, SizeConverter.P3String, SizeConverter.P4String, SizeConverter.P5String,
```
Comment (cormacpayne) [inline src/ResourceManager/RedisCache/Commands.RedisCache/Commands/NewAzureRedisCache.cs]: @SiddharthChatrolaMs any test functionality for the new values to ensure that they work on the server?

Comment (SiddharthChatrolaMs) [reply src/ResourceManager/RedisCache/Commands.RedisCache/Commands/NewAzureRedisCache.cs]: We wanted to make this parameter string so whenever we have new Sku, customers can create new caches without PowerShell update. Existing tests would be enough for that. As we don't test all of the values provided here

File: src/ResourceManager/RedisCache/ChangeLog.md (hunk 1 above)
Comment (SiddharthChatrolaMs) [reply src/ResourceManager/RedisCache/ChangeLog.md]: This is fixed

**Comment by SiddharthChatrolaMs** [comment]
@cormacpayne Fixed one comment and replied to another one

**Pull request merged**